README.md

```diff
@@ -28,11 +28,13 @@ A collection of our Android templates:
     app-name=                          New app name (i.e., MyApp, "My App", "my-app")
     template=                          Template (i.e., xml, compose)
     force=                             Force project creation even if the script fails (default: false)
+    destination=                       Set the output location where the project should be generated (i.e., /Users/johndoe/documents/projectfolder)
   ```
 
   Examples: 
     `kscript new_project.kts package-name=co.myxmlproject.example app-name="My XML Project" template=xml`
     `kscript scripts/new_project.kts package-name=co.myxmlproject.example app-name="My XML Project" template=xml`
+    `kscript new_project.kts package-name=co.nimblehq.suvkmmsurveys.android app-name=Surveys template=compose destination=/Users/johndoe/documents/projectfolder`
```

Comment:
Shall we update this to match the other examples? 😄 
```suggestion
    `kscript new_project.kts package-name=co.myxmlproject.example app-name="My XML Project" template=xml destination=/Users/johndoe/documents/projectfolder`
```

Reply:
Updated in https://github.com/nimblehq/android-templates/pull/477/commits/1eeb2103166dbe93e9f25f2013b5c34fbe6a5e1a